cypress/integration/generic_templates.ts

```diff
@@ -314,7 +314,7 @@ context('GenericTemplates tests', () => {
       cy.contains(genericTemplateQuestions[1]).should('exist');
     });
 
-    it('Should be able to create proposal with genericTemplate', () => {
+    it.only('Should be able to create proposal with genericTemplate', () => {
```

Comment:
You accidentally left the `only` in here 🙂

Reply:
Resolved! Thank you :)